CHANGELOG.md

```diff
@@ -4,6 +4,9 @@
 
 ## 0.11.0 (2023-01-31)
 
+- [API Change] Adds `plot.CalibrationCurve` object-oriented API
+- [Doc] User Guide on Probability Calibration
```

Comment:
changelog notes should always go in the top section (development version), please update

Reply:
I think it happened because of a version bump before the review